### ISSUE TITLE
Don't break JVM/JS builds if Java 7

### ISSUE THREAD

**Issue by xuwei-k** [scala-native/scala-native#673]
similar to https://github.com/scala-native/scala-native/issues/562

I just tried update scala-native 0.2.0 in scalaz `series/7.2.x` branch (still support Java 7). but I got following error while loading project

- https://github.com/xuwei-k/scalaz/commit/2ad1d4efef99230f59a70498be69db6e186b857d
- https://travis-ci.org/xuwei-k/scalaz/jobs/226224759#L783

```
java.lang.RuntimeException: Java 8 or newer is required for this project.
	at scala.sys.package$.error(package.scala:27)
	at scala.scalanative.sbtplugin.ScalaNativePluginInternal$$anonfun$unscopedSettings$2.apply(ScalaNativePluginInternal.scala:113)
	at scala.scalanative.sbtplugin.ScalaNativePluginInternal$$anonfun$unscopedSettings$2.apply(ScalaNativePluginInternal.scala:110)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.EvaluateSettings$MixedNode.evaluate0(INode.scala:175)
	at sbt.EvaluateSettings$INode.evaluate(INode.scala:135)
	at sbt.EvaluateSettings$$anonfun$sbt$EvaluateSettings$$submitEvaluate$1.apply$mcV$sp(INode.scala:69)
	at sbt.EvaluateSettings.sbt$EvaluateSettings$$run0(INode.scala:78)
	at sbt.EvaluateSettings$$anon$3.run(INode.scala:74)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
[error] Java 8 or newer is required for this project.
[error] Use 'last' for the full log.
Project loading failed: (r)etry, (q)uit, (l)ast, or (i)gnore? 
```

**Comment by densh** [comment]
https://github.com/scala-native/scala-native/pull/674 replaces error with warning if older version of JVM is detected. Nevertheless I'm a bit surprised that it used to work before, given that Scala Native uses JDK8+ APIs in the toolchain. 

We're going to release a hotfix release (`0.2.1`) in next few days to address this breakage.

**Comment by densh** [comment]
I've just published 0.2.1 with a fix for this issue, @xuwei-k can you please confirm that it indeed solves your problem before we announce it publicly?

**Comment by xuwei-k** [comment]
I have confirmed. Thank you for quick fix 👍
